### PR TITLE
Redesign: Media edit mode resize

### LINKED ISSUE #6202: Redesign: Media edit mode
## Feature Description

Update design for media editor based on [Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=1009%3A4323).

- [x] Update selection handle and border size and shape
~-  Update slider using design system~ WILL BE DONE SEPARATELY, dependent on #4938 

| Current design | New design |
|-|-|
| Screen Shot 2021-02-04 at 22 30 54 | Screen Shot 2021-02-04 at 22 30 59 |

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## To-do

- [x] Verify if it's OK to separate slider component into a separate issue (creating the component + switching to the new component in the edit mode).

## User-facing changes
The Media edit mode resize has been redesigned in this PR. Please see [Figma.](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=3616%3A64347)

Note: this PR does NOT include slider update since that component has not been created yet.

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Add a media element
2. Enter edit mode
3. Verify that the resize box looks like in Figma (see link above).

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6202

### CODE REVIEW

File: assets/src/edit-story/components/moveable/cropStyle.js
Comment (barklund) [inline assets/src/edit-story/components/moveable/cropStyle.js]: Should be the new `border.selection` too when implemented (ping @BrittanyIRL).

File: assets/src/edit-story/components/moveable/cropStyle.js
Comment (barklund) [inline assets/src/edit-story/components/moveable/cropStyle.js]: Ditto

File: assets/src/edit-story/components/moveable/cropStyle.js
Comment (barklund) [inline assets/src/edit-story/components/moveable/cropStyle.js]: This doesn't look correct. It's close, but not really there - two corners are missing rounding and the inner corners (on the shape itself, not the border) are not rounded at all.

Let's change it to 3 shapes - the primary element and both `::before` and `::after`, styled like this:
Screen Shot 2021-02-08 at 19 09 24

I've tried styling the corners using this idea in c0bd7a6f8, and I like it (well, better at least).

| Design | Clip-path | `::before` + `::after` |
|-|-|-|
| Screen Shot 2021-02-08 at 19 59 54 | Screen Shot 2021-02-08 at 19 59 41 | Screen Shot 2021-02-08 at 20 01 41 |

Yes yes, I know. It's tiny details that nobody will notice. But I did! 😁

**Review by barklund — COMMENTED** [review]
Added an update for nicer corners just to make things more complex 🤷  - and a few comments.

**Comment by barklund** [comment]
@googlebot I consent.

File: assets/src/edit-story/components/moveable/cropStyle.js
Comment (BrittanyIRL) [inline assets/src/edit-story/components/moveable/cropStyle.js]: 🎉

**Review by barklund — APPROVED** [review]
Good to go after #6209 is merged in.

**Pull request merged**
